maliput/src/api/mod.rs

```diff
@@ -1353,7 +1366,7 @@ impl<'a> IntersectionBook<'a> {
     ///   * `id` - The id of the Intersection to get.
     ///
     /// ## Returns
-    ///   * An Option<Intersection>
+    ///   * An `Option<Intersection>`
```

Comment:
What about the next lines? Shouldn't we use \` (backticks) as well?

Reply:
we should use backticks. If we don't have backticks a warning is raised by cargo doc command:

![image](https://github.com/maliput/maliput-rs/assets/53065142/db249c64-8d1c-4772-af99-1d4b80a2ea98)

I should have added this change to a separate PR tho.